Makefile

```diff
@@ -1,13 +1,16 @@
+# Setting SHELL to bash allows bash commands to be executed by recipes.
+# Options are set to exit when a recipe line exits non-zero or a piped command fails.
+SHELL = /usr/bin/env bash -o pipefail
+.SHELLFLAGS = -ec
 
 # Image URL to use all building/pushing image targets
 IMG ?= numaplane-controller
 VERSION ?= latest
```

Comment:
What is the difference between `VERSION` and `BASE_VERSION`?

Reply:
would it make sense to add a comment for it then?

Reply:
Added a short comment for that.